### DISCUSSION TITLE
Snapcast Client Multiple Instances

### DISCUSSION THREAD

**Discussion by SweatyToast** [snapcast/snapcast#1118] [General]
I'm seeking assistance to achieve my objective of creating a  virtual machine with multiple instances of Snapcast Client, including ones for my game room, master bedroom, bathroom, kitchen, and notification sounds. Currently, I'm using a Behringer UMC1820 in Jack audio, with separate sinks for each room to transmit audio successfully. However, 
I'm encountering difficulties connecting my Snapcast client instances to the sink device because JACK may not be supported. 

If I do a Snapcast -l I don't see my JACK sinks

Can anyone suggest a straightforward method to accomplish this?

**Comment by SweatyToast** [comment]
I tried like doing

snapclient --player pulse:property=device=game_room 

but no dice. Is there a way to specify a device. When I do this it just states its using the default because no parameters are specified.

**Reply by ConnorsApps** [reply]
I'm not using pulse, but one of the options to run multiple clients on one host is by using the `--instance int` flag.
